kolibri/core/assets/src/views/CoreSnackbar/index.vue

```diff
@@ -108,23 +108,12 @@
 
 <style lang="scss" scoped>
 
-  @import '~kolibri.styles.definitions';
-
   .snackbar {
     position: fixed;
     bottom: 0;
     left: 0;
     z-index: 24;
     margin: 16px;
-
-    /deep/ .ui-snackbar__action-button {
-      font-weight: bold;
-      color: $core-bg-light;
-    }
-
-    /deep/ .ui-button--type-secondary:hover:not(.is-disabled) {
-      background-color: rgba(255, 255, 255, 0.05);
-    }
```

Comment:
Looks like we lost some styles (regression of #2798):

![image](https://user-images.githubusercontent.com/2367265/50176157-84af6700-02b3-11e9-8b39-be1cd48975e0.png)